source/streaming-mode/streaming-read-config.txt

```diff
@@ -203,4 +207,92 @@ You can configure the following properties when reading a change stream from Mon
+configuration property by separating the collection names
+with a comma. Do not add a space between the collections unless the space is a
+part of the collection name.
+
+Specify multiple collections as shown in the following example:
+
+.. code-block:: java
+
+   ...
+   .option("spark.mongodb.collection", "collectionOne,collectionTwo")
+
+If a collection name is "*", or if the name includes a comma or a backslash (\\),
+you must escape the character as follows:
+
+- If the name of a collection used in your ``collection`` configuration
+  option contains a comma, the {+connector-short+} treats it as two different
+  collections. To avoid this, you must escape the comma by preceding it with
+  a backslash (\\). Escape a collection named "my,collection" as follows:
+
+  .. code-block:: java
+
+     "my\,collection"
+
+- If the name of a collection used in your ``collection`` configuration
+  option is "*", the {+connector-short+} interprets it as a specification
+  to scan all collections. To avoid this, you must escape the asterisk by preceding it
+  with a backslash (\\). Escape a collection named "*" as follows:
+
+  .. code-block:: java
+
+     "\*"
+
+- If the name of a collection used in your ``collection`` configuration
+  option contains a backslash (\\), the
+  {+connector-short+} treats the backslash as an escape character, which
+  might change how it interprets the value. To avoid this, you must escape
+  the backslash by preceding it with another backslash. Escape a collection named "\\collection" as follows:
+
+  .. code-block:: java
+
+     "\\collection"
+  
+  .. note:: 
+     
+     When specifying the collection name as a string literal in Java, you must
+     further escape each backslash with another one. For example, escape a collection 
+     named "\\collection" as follows:
+
+     .. code-block:: java
+
+        "\\\\collection"
+
+You can stream from all collections in the database by passing an
+asterisk (*) as a string for the collection name.
+
+Specify all collections as shown in the following example:
+
+.. code-block:: java
+
+   ...
+   .option("spark.mongodb.collection", "*")
+
+If you create a collection while streaming from all collections, the new
+collection is automatically included in the stream. 
+
+You can drop collections at any time while streaming from multiple collections.
+
+.. important:: Inferring the Schema of a Change Stream
+
+   If you set the ``change.stream.publish.full.document.only``
+   option to ``true``, the {+connector-short+} infers the schema of a ``DataFrame``
+   by using the schema of the scanned documents. If you set the option to
+   ``false``, you must specify a schema.
+   
+   Schema inference happens at the beginning of streaming, and does not take into
+   account collections that are created during streaming.
+
+   When streaming from multiple collections, the connector samples
+   each collection sequentially. Streaming from a large number of
```

Comment:
[optional]

My original wording in the "Documentation Changes Summary" was proven to be unclear, and now I am afraid to leave users confused. I am suggesting the following additions:

```suggestion
   When streaming from multiple collections, and inferring the schema,
   the connector samples each collection sequentially
   as part of the schema inference. Streaming from a large number of
```

Reply:
Added to this to make it extra clear